README.md

```diff
@@ -26,6 +26,20 @@ plugins =
 
 in your `mypy.ini` file.
 
+## FAQ
+
+### Model instance is inferred as `Any` instead of my `Model` class
+
+This library adds support for [Generics](https://peps.python.org/pep-0484/#generics) in DRF. For example, you can now write:
+
+```python
+class MyModelSerializer(serializers.ModelSerializer[MyModel]):
```

Comment:
Do we need to monkeypatch it?

Reply:
Do you mean call `django_stubs_ext.monkeypatch()`? I'm pretty sure it's not needed, worked without it in my project.

Reply:
The `__class_getitem__` method was added https://github.com/encode/django-rest-framework/pull/7385 which was released in DRF 3.12